package.json

```diff
@@ -0,0 +1,251 @@
+{
+  "name": "no-name",
+  "license": "UNLICENSED",
```

Comment:
Shouldn't we put something proper in here?

Reply:
I guess we can but this is normally for producing an npm package which we will never really do with this.